### PR TITLE
[CHORE] Log Salesforce API responses in the debug mode

### LINKED ISSUE #1303: [SF] Log SF API responses in the debug mode which will help in debugging

### PR DESCRIPTION
**Pull request by Shailesh351**
Closes https://github.com/WideChat/Rocket.Chat/issues/1303

### CODE REVIEW

**Comment by ear-dev** [comment]
@AlexanderKanakis can you review please?  thanks.

**Comment by ear-dev** [comment]
@csmith14 any chance you can review this PR please?  thanks...

File: SalesforceLiveAgentApp.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export class SalesforceLiveAgentApp
 		super(info, logger, accessors);
 	}
 
-	public async initialize(configurationExtend: IConfigurationExtend, environmentRead: IEnvironmentRead): Promise<void> {
+	public async initialize(configurationExtend: IConfigurationExtend, _environmentRead: IEnvironmentRead): Promise<void> {
```
Comment (csmith14) [inline SalesforceLiveAgentApp.ts:53]: Why have we started replacing some of these parameter names with leading underscores? I've noticed it in a few PRs and would like to know why, so I can follow whatever the new convention is.

**Review by csmith14 — APPROVED** [review]

File: SalesforceLiveAgentApp.ts (hunk 1 above)
Comment (AlexanderKanakis) [reply SalesforceLiveAgentApp.ts:53]: @csmith14  Underscores indicate function parameters that need to exist but are not used within the function block. We avoid TSlint issues that way.

**Pull request merged**